### PR TITLE
Sampler smoke

### PR DESCRIPTION
**Pull request by denicaM**

### CODE REVIEW

File: test/smoke/sampler/src/test/test.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    await samplerClient.stopSampling(eventBuilder.createStopEvent(moment.now()));
+  });
+
+  it('samples usage to Abacus successfully', async () => {
+    log('Sending start event to sampler ...');
+    response = await samplerClient.startSampling(eventBuilder.createStartEvent(moment.now()));
+    expect(response.statusCode).to.be.equal(httpStatus.CREATED);
+
+    log('Sending stop event to sampler ...');
+    response = await samplerClient.stopSampling(eventBuilder.createStopEvent(moment.now()));
```
Comment (momchil-sap) [inline test/smoke/sampler/src/test/test.js]: I still think that you should ensure that the start and stop moments are not the same.

File: test/smoke/sampler/package.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
+{
+  "name": "abacus-sampler-smoke-test",
+  "description": "Saampler smoke test",
```
[the remark this answers is not in the document]
Comment (denicaM) [reply test/smoke/sampler/package.json]: Done

File: test/smoke/sampler/src/test/test.js [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+  after(async() => {
+    log('Stopping sampling in case of test error ...');
+    await stopSamplingGracefully(receiverClient, eventBuilder.createStopEvent(moment.now()));
+  });
+
+  it('samples usage to Abacus successfully', async () => {
+    log('Creating services mappings ...');
+    await createMappingGracefully(receiverClient, mapping);
+
+    log('Cleaning up in case of previous test error ...');
```
Comment (momchil-sap) [inline test/smoke/sampler/src/test/test.js]: Shouldn't this be in the `before` or `beforeEach` section? If we ever do another `it` we won't need to copy-paste it.

Comment (denicaM) [reply test/smoke/sampler/src/test/test.js]: New context with `beforeEach` was pushed

**Pull request merged**